zip-bolt-support.rst

```diff
@@ -0,0 +1,334 @@
+::
+
+  ZIP: XXX
+  Title: Add support for Blind Off-chain Lightweight Transactions (Bolt) protocol
+  Authors: J. Ayo Akinyele <ayo@boltlabs.io>
+           Colleen Swanson <swan@boltlabs.io>
+  Credits: Ian Miers <imiers@z.cash>
+           Matthew Green <mgreen@z.cash>
+  Category: Consensus
```

Comment:
```suggestion
  Category: Informational
```
This is an exploration of the design space, so it should be Informational. If it is distilled down to a specific design, we can change the category.